src/components/modal/report/modal.jsx

```diff
@@ -6,95 +6,50 @@ const FormattedMessage = require('react-intl').FormattedMessage;
 const injectIntl = require('react-intl').injectIntl;
 const intlShape = require('react-intl').intlShape;
 const Modal = require('../base/modal.jsx');
-const classNames = require('classnames');
 
 const ModalTitle = require('../base/modal-title.jsx');
 const ModalInnerContent = require('../base/modal-inner-content.jsx');
-const Form = require('../../forms/form.jsx');
-const Button = require('../../forms/button.jsx');
 const Select = require('../../forms/select.jsx');
-const Spinner = require('../../spinner/spinner.jsx');
 const TextArea = require('../../forms/textarea.jsx');
-const FlexRow = require('../../flex-row/flex-row.jsx');
 const previewActions = require('../../../redux/preview.js');
+const Progression = require('../../progression/progression.jsx');
+const FormStep = require('./form-step.jsx');
+const {reportOptionsShape, REPORT_OPTIONS} = require('./report-options.js');
 
 require('../../forms/button.scss');
 require('./modal.scss');
 
-const REPORT_OPTIONS = [
-    {
-        value: '',
-        label: {id: 'report.reasonPlaceHolder'},
-        prompt: {id: 'report.promptPlaceholder'}
-    },
-    {
-        value: '0',
-        label: {id: 'report.reasonCopy'},
-        prompt: {id: 'report.promptCopy'}
-    },
-    {
-        value: '1',
-        label: {id: 'report.reasonUncredited'},
-        prompt: {id: 'report.promptUncredited'}
-    },
-    {
-        value: '2',
-        label: {id: 'report.reasonScary'},
-        prompt: {id: 'report.promptScary'}
-    },
-    {
-        value: '3',
-        label: {id: 'report.reasonLanguage'},
-        prompt: {id: 'report.promptLanguage'}
-    },
-    {
-        value: '4',
-        label: {id: 'report.reasonMusic'},
-        prompt: {id: 'report.promptMusic'}
-    },
-    {
-        value: '8',
-        label: {id: 'report.reasonImage'},
-        prompt: {id: 'report.promptImage'}
-    },
-    {
-        value: '5',
-        label: {id: 'report.reasonPersonal'},
-        prompt: {id: 'report.promptPersonal'}
-    },
-    {
-        value: '6',
-        label: {id: 'general.other'},
-        prompt: {id: 'report.promptGuidelines'}
-    }
-];
+// Progression component only addresses steps by number, but this flow
```

Comment:
A little hard to read this comment. How about:

// The Progression component uses numbers to track which step it's on, but that's
// hard to read. Make the code easier to read by giving each step number a readable label.

Reply:
Done